locales/pl.json

```diff
@@ -1,155 +1,164 @@
 {
-	"Notes": "Notes",
-	"Tags": "Tags",
-	"Preferences": "Preferences",
-	"Make a note": "Make a note",
+	"Notes": "Notatki",
+	"Tags": "Tagi",
+	"Preferences": "Ustawienia",
+	"Make a note": "Stwórz notatkę",
 	"Ctrl": "Ctrl",
 	"Ctrl(^)": "Ctrl",
-	"to create a new note": "to create a new note",
-	"Toggle Mode": "Toggle Mode",
-	"Trash": "Trash",
-	"MODIFICATION DATE": "MODIFICATION DATE",
-	"Words": "Words",
-	"Letters": "Letters",
-	"STORAGE": "STORAGE",
+	"to create a new note": "Aby stworzyć nową notatkę",
+	"Toggle Mode": "Przełącz tryb",
+	"Trash": "Kosz",
+	"MODIFICATION DATE": "DATA MODYFIKACJI",
+	"Words": "Słowa",
+	"Letters": "Litery",
+	"STORAGE": "MIEJSCE ZAPISU",
 	"FOLDER": "FOLDER",
-	"CREATION DATE": "CREATION DATE",
-	"NOTE LINK": "NOTE LINK",
+	"CREATION DATE": "DATA UTWORZENIA",
+  "NOTE LINK": "LINK NOTATKI",
+  "Toggle editor mode": "Przełączanie trybu edytora",
 	".md": ".md",
 	".txt": ".txt",
 	".html": ".html",
-	"Print": "Print",
-	"Your preferences for Boostnote": "Your preferences for Boostnote",
+  "Print": "Drukuj",
+  "Help": "Pomoc",
+	"Your preferences for Boostnote": "Twoje ustawienia dla Boostnote",
 	"Storages": "Storages",
-	"Add Storage Location": "Add Storage Location",
-	"Add Folder": "Add Folder",
-	"Open Storage folder": "Open Storage folder",
-	"Unlink": "Unlink",
-	"Edit": "Edit",
-	"Delete": "Delete",
-	"Interface": "Interface",
-	"Interface Theme": "Interface Theme",
-	"Default": "Default",
-	"White": "White",
+	"Add Storage Location": "Dodaj miejsce zapisu",
+	"Add Folder": "Dodaj Folder",
+	"Open Storage folder": "Otwórz folder zapisu",
+	"Unlink": "Odlinkuj",
+	"Edit": "Edytuj",
+	"Delete": "Usuń",
+	"Interface": "Interfejs",
+	"Interface Theme": "Styl interfejsu",
+	"Default": "Domyślny",
+	"White": "Biały",
 	"Solarized Dark": "Solarized Dark",
-	"Dark": "Dark",
-	"Show a confirmation dialog when deleting notes": "Show a confirmation dialog when deleting notes",
-	"Editor Theme": "Editor Theme",
-	"Editor Font Size": "Editor Font Size",
-	"Editor Font Family": "Editor Font Family",
-	"Editor Indent Style": "Editor Indent Style",
-	"Spaces": "Spaces",
-	"Tabs": "Tabs",
-	"Switch to Preview": "Switch to Preview",
-	"When Editor Blurred": "When Editor Blurred",
-	"When Editor Blurred, Edit On Double Click": "When Editor Blurred, Edit On Double Click",
-	"On Right Click": "On Right Click",
-	"Editor Keymap": "Editor Keymap",
-	"default": "default",
+	"Dark": "Ciemny",
+  "Show a confirmation dialog when deleting notes": "Zatwierdzaj usunięcie notatek",
+  "Show only related tags": "Pokazuj tylko powiązane tagi",
+	"Editor Theme": "Wygląd edytora",
+	"Editor Font Size": "Rozmiar czcionki",
+  "Editor Font Family": "Czcionka",
+  "Snippet Default Language": "Domyślny język snippetów kodu",
+	"Editor Indent Style": "Rodzaj wcięć",
+	"Spaces": "Spacje",
+	"Tabs": "Tabulatory",
+	"Switch to Preview": "Przełącz na podgląd",
+	"When Editor Blurred": "Gdy edytor w pracuje w tle",
```

Comment:
"Gdy edytor w pracuje w tle" -> "Gdy edytor stracił fokus"
Yes, "stracił fokus" is not a beautiful Polish , but the editor is not "working" in the background, it just is there.
So maybe "Gdy edytor jest w tle" ? 

But generally I don't urge on this one